v0/destinations/attribution/transform.js

```diff
@@ -40,6 +40,7 @@ function getTransformedJSON(message) {
     ? message.properties
     : undefined;
   const event = get(message, "event") ? message.event : undefined;
```

Comment:
```suggestion
  const event = get(message, "event")
```
get will return undefined if `event` is not presnet